tests/Integration/GeocoderTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Exceptions\InvalidAddressGeocodingException;
+use App\Services\Geocoder;
+use Illuminate\Support\Facades\Http;
+use Tests\TestCase;
+
+class GeocoderTest extends TestCase
```

Comment:
Will these tests fail if someone doesn't have a `GOOGLE_API_KEY` in their `.env`?

Reply:
Yes, it will fail without the key although it won't run as part of the whole test suite since I moved it to `tests/Integration`.  I was thinking this would be a test that rarely needs to run since its primary purpose is to ensure we are calling Google's API correctly.  Ideally we'd run this as part of the Github action but without exposing the key.  I made a Trello card to investigate options for this.